src/modules/Api.js

```diff
@@ -166,8 +200,17 @@ export default class Api {
           'Content-type': 'application/json; charset=UTF-8',
         },
       },
+    );
+    this.DisplayComm(data.item_id);
+  };
+
+  DisplayComm = async (data) => {
+    await fetch(
+      `${this.InvolvementApiEP}apps/${this.InvolvementAppID}/comments?item_id=${data}`,
     )
       .then((response) => response.json())
-      .then((json) => json);
+      .then((json) => {
+        DisplayComments(json);
+      });
```

Comment:
there is an error when the item has no comment yet and its because the json is not returned as array , that will cause error in
https://github.com/rbhatt1999/JavaScript-Capstone/blob/31022ee589773784015de38d2dc80021731fa49c/src/modules/Functions.js#L105-L110

so kindly fix that so you get rid of that problem
suggestion :
you can add a conditional statement when you get JSON as response and check if its an array with a minimum length of 1

Reply:
I've implemented the requested changes